src/components/maprecorder/MapRecorderWin.vue

```diff
@@ -0,0 +1,299 @@
+              <v-slider
+                  prepend-icon="mdi-iframe-variable-outline"
+                  v-model.number="frameRate"
+                  min="20"
+                  max="50"
+                  step="1"
+                  thumb-label
+                  hide-details>
+              </v-slider>
+            </v-card-text>
+
+            <v-subheader>Bit rate (MBits/s)</v-subheader>
+            <v-card-text class="pt-0">
+              <v-slider
+                  prepend-icon="mdi-quality-high"
+                  v-model.number="videoMBitsPerSecond"
+                  min="1.0"
+                  max="10.0"
+                  step="0.1"
+                  thumb-label
+                  hide-details>
+              </v-slider>
+            </v-card-text>
+
+            <v-subheader>Filename</v-subheader>
+            <v-card-text class="pt-0">
+              <v-text-field
+                v-model="filename"
+                prepend-icon="mdi-rename-box"
+                label="YYYY-MM-DD at HH.MM.SS"
+                dense
+                single-line
+                hide-details
+              ></v-text-field>
+            </v-card-text>
+          </v-card>
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
+
+    <v-card-actions>
+      <v-layout align-center>
+        <v-btn
+          class="mr-3"
+          icon 
+          fab
+          outlined 
+          @click="toggleRecord">
+          <v-icon v-if="recording">mdi-video-off</v-icon> 
+          <v-icon v-else>mdi-video</v-icon> 
+        </v-btn>
+        <v-flex fill-height grow> 
+          <v-progress-linear
+            :active="recording"
+            buffer-value="0"
+            stream
+          ></v-progress-linear>
+        </v-flex>
+        <v-flex fill-height shrink>
+           <v-alert
+            v-model="error"
+            type="error" 
+            dismissible>
+            Failed to start recording.
+          </v-alert>
+        </v-flex>
+      </v-layout>
+    </v-card-actions>
+  </wgu-module-card>
+</template>
+
+<script>
+import ModuleCard from './../modulecore/ModuleCard';
+import { Mapable } from '../../mixins/Mapable';
+import createCanvasRecorder from 'canvas-record';
+
+export default {
+  name: 'wgu-maprecorder-win',
+  inheritAttrs: false,
+  mixins: [Mapable],
+  components: {
+    'wgu-module-card': ModuleCard
+  },
+  props: {
+    icon: {type: String, required: false, default: 'mdi-video'},
+    title: {type: String, required: false, default: 'Map recorder'}
+  },
+  data () {
+    const mimeTypes = this.getSupportedMimeTypes();
+
+    return {
+      moduleName: 'wgu-maprecorder',
+
+      /**
+      * Custom canvas element for drawing the Open layers map.
+      */
+      mapCanvas: null,
+      /**
+       * Context of the custom canvas element.
+       */
+      mapContext: null,
+      /**
+       * Canvas recorder.
+       */
+      recorder: null,
+      /**
+       * Recording state
```

Comment:
```suggestion
       * Recording state.
```

Reply:
Thanks for the review Jakob! I'll hopefully find some time this week to look into the requested changes.

Reply:
@JakobMiksch the points I listed are just ideas that we can convert to future issues
I am totally fine with merging this PR as is

Reply:
I suggest you wait with merging, since fixing a few formatting issues isn't really worth a separate PR. Regarding your proposals I'll have a look and see what is easy to achieve.

Reply:
I will not merge anyway. let's wait for @chrismayer